cmd/protecodeExecuteScan.go

```diff
@@ -336,5 +336,10 @@ func getTarName(config *protecodeExecuteScanOptions) string {
 	}
 	// replace unwanted chars
 	fileName = strings.ReplaceAll(fileName, "/", "_")
+	sha256 := "@sha256"
+	if strings.Contains(fileName, sha256) {
+		fileName = strings.ReplaceAll(fileName, ":", "_")
+		fileName = strings.ReplaceAll(fileName, sha256, "")
```

Comment:
So what is the result? I would like to have a unit test with a practical example for this, so I can picture it.

Reply:
unit test can be found here : 
[Unit Test](https://github.com/SAP/jenkins-library/pull/2473/files#diff-c1bb3605bd36b521f5c3831dcb2f44793851a5dca8a58ea0f98dbc0275f2c21eR407)